src/views/Error/Error.vue

```diff
@@ -19,7 +19,7 @@ export default defineComponent({
   setup() {
     const error = store.getters.error.message;
     const isInDevMode =
-      process.env.NODE_ENV == "test" || process.env.NODE_ENV == "development"
+      process.env.NODE_ENV === "test" || process.env.NODE_ENV === "development"
```

Comment:
I think we would want a test environment to act as production, so only checking for "development" might be the way to go. (apologies if I told you conflicting information)

Reply:
fixed